### ISSUE TITLE
JK's Skyrim compatibility patches with THC Dawnstar / Winterhold not recognized

### ISSUE THREAD

**Issue by suamor** [loot/skyrimse#2500]
Installed: [JK's Sykrim](https://www.nexusmods.com/skyrimspecialedition/mods/6289)
Installed: [The Great City of Dawnstar](https://www.nexusmods.com/skyrimspecialedition/mods/19491)
Installed: [The Great City of Winterhold](https://www.nexusmods.com/skyrimspecialedition/mods/17127)
Installed: [The great Cities of JK's North](https://www.nexusmods.com/skyrimspecialedition/mods/33687)
Also installed: Rob's compatibility patches for [Dawnstar](https://www.nexusmods.com/skyrimspecialedition/mods/66656 ) and [Winterhold](https://www.nexusmods.com/skyrimspecialedition/mods/65456). They contain TGC fixes. Do not let yourself confuse by the name of the mod.
RED marked: these are not compatible
Loot does not recognize that I have installed the compatibility patches.

**Comment by MacSplody** [comment]
```yaml
  - name: 'JKs Skyrim.esp'
    inc:
      - name: 'The Great Cities - Minor Cities and Towns.esp'
        condition: 'checksum("JKs Skyrim.esp", C8478AEC) or not active("The Great Cities of JK''s (North|Skyrim) - Patch.esp")'
      - 'The Great City of Dawnstar.esp'
      - 'The Great City of Falkreath.esp'
      - 'The Great City of Morthal.esp'
      - 'The Great City of Winterhold.esp'
```

Right those would need to be updated to check for the patches.

**Comment by suamor** [comment]
Ok, the issue was that 'The Great Cities - Minor Cities and Towns' is a merger of all these mods (and more, see mod description). 
So the warning is somewhat correct about unecessary plugin, but not because of JKs Skyrim but because it is already installed (via merger), so it can be safely removed.

**Comment by sibir-ine** [comment]
I added messages for the mods included in TGC - Minor Cities & Towns in #2520, so that should help avoid confusion in the future.
